docs/docs/providers/shopify.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Let's make sure that this documentation page has the same structure as others, with the following headings:

- Documentation
- Configuration
- Options
- Example